src/applications/check-in/day-of/sass/check-in.scss

```diff
@@ -77,4 +77,16 @@ ol.appointment-list{
       }
     }
   }
+}
+
+.check-in-error-arrow {
+  margin: 1.5rem 0;
```

Comment:
If possible, please use VADS classes instead of creating a new one (`vads-u-margin-...`)